### PR TITLE
Adding SOT363

### LINKED ISSUE #44: SOT363
https://www.nexperia.com/packages/SOT363.html

### PR DESCRIPTION
**Pull request by DhairyaMajmudar**
-> Adding SOT363 circuit
-> Closes #44 

cc: @seveibar

### CODE REVIEW

File: src/fn/index.ts [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
+export { sot363 } from "./sot363";
+export { sot23 } from "./sot23";
+export { dfn } from "./dfn";
+export { pinrow } from "./pinrow";
+export { sot563 } from "./sot563";
+export { ms012 } from "./ms012";
+export { ms013 } from "./ms013";
+export { sot723 } from "./sot723";
+export { sod123 } from "./sod123";
+export { axial } from "./axial";
```
Comment (seveibar) [inline src/fn/index.ts:24]: hmm this file was reformatted with what looks like prettier, all tscircuit projects use biome though, can we reformat? I will add a github workflow to check for this in the near future

**Review by seveibar — CHANGES_REQUESTED** [review]
the svg shows the footprint is incorrect

image

File: tests/sot363.test.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import test from "ava";
+import { getTestFixture } from "tests/fixtures/get-test-fixture";
+
+test("sot363", async (t) => {
+  const { fp, logSoup, snapshotSoup } = await getTestFixture(t);
+  const soup = fp.string("sot363").soup();
+
+  await logSoup(soup);
```
Comment (seveibar) [inline tests/sot363.test.ts:8]:
```suggestion
```

File: src/fn/index.ts (hunk 1 above)
Comment (DhairyaMajmudar) [reply src/fn/index.ts:24]: yeah sure, do I undo the formating in this PR? or leave it as is ?

Comment (DhairyaMajmudar) [reply src/fn/index.ts]: Yep, I've removed the prettier formatting

**Review by seveibar — APPROVED** [review]
nice

**Pull request merged**